### ISSUE TITLE
dart all pushed in a single line

### ISSUE THREAD

**Issue by gnunicorn** [udoprog/genco#41]
using genco 0.17.5 the default formatter for `dart` appears to be broken - it puts everything into a single line. Which in turn breaks dartc when having `#[doc = ` in there.

Take this cargo-play example:
```rust
//# genco = "0.17.5"

use genco::prelude::*;

fn main() {
    // Some JSON input data as a &str. Maybe this comes from the user.
    let tokens: dart::Tokens = quote! {
        /// doc comment
        class Api {
            final String lateString;
        }
    };

    assert_eq!(tokens.to_file_vec().unwrap().len(), 4);
}

```

Rather than 4 lines, `to_file_vec` wraps it all into one :( .

My cargo: `cargo 1.70.0 (ec8a8a0ca 2023-04-25)` but nightly doesn't show anything different.

**Comment by udoprog** [comment]
See #39, which should hopefully be a temporary issue.

And as for the comment, it's expanded as a attribute before it's handed to the macro, so you have to do something like this for verbatim comments:

```rust
    let tokens: dart::Tokens = quote! {
        $("/// doc comment")
        class Api {
            final String lateString;
        }
    };
```
I usually end up writing a helper [like this](https://github.com/reproto/reproto/blob/b698923652bae49812f4becb505898b4fb6b4268/lib/backend-rust/src/utils.rs#L5) if I'm dealing with comment heavy code generation.

**Comment by gnunicorn** [comment]
thanks, for the quick feedback @udoprog . Indeed forcing proc-macro to be at 1.0.55 (by using `
proc-macro2 = "=1.0.55"` in my `Cargo.toml`) fixes the multi-line issue. Your example for docs also works well. Thanks!

feel free to close this as a duplicate of #39 then...

**Comment by udoprog** [comment]
Fix for #39 is incoming in #43.
